src/coprocessor/dag/expr/fncall.rs

```diff
@@ -210,7 +210,14 @@ impl FnCall {
 
             ScalarFuncSig::JsonExtractSig |
             ScalarFuncSig::JsonRemoveSig |
-            ScalarFuncSig::JsonMergeSig => (2, usize::MAX),
+            ScalarFuncSig::JsonMergeSig |
+            ScalarFuncSig::InInt |
+            ScalarFuncSig::InReal |
+            ScalarFuncSig::InString |
+            ScalarFuncSig::InDecimal |
+            ScalarFuncSig::InTime |
+            ScalarFuncSig::InDuration |
+            ScalarFuncSig::InJson => (2, usize::MAX),
```

Comment:
It seems that `select NULL in (select * from test.test)s` will get 0 if `test.test` is empty.

Reply:
This case won't affect tikv's part.